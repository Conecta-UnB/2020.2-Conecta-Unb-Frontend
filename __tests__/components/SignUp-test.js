import React from 'react';
import { render } from '@testing-library/react';
import SignUpForm from '../../src/components/Forms/SignUp';

describe('testando loginforms', () => {
  test('deve ser igual a snapshot', () => {
    const { asFragment } = render(<SignUpForm />);
    expect(asFragment(<SignUpForm />)).toMatchSnapshot();
  });
});
