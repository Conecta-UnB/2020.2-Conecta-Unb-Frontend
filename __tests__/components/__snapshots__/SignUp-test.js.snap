// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando loginforms deve ser igual a snapshot 1`] = `
<DocumentFragment>
  <div
    class="containerSignUpForm"
  >
    <form
      action="#"
    >
      <input
        name="nome"
        value="Nome completo"
      />
      <input
        name="matricula"
        value="MatrÃ­cula"
      />
      <input
        name="email"
        type="email"
        value="E-mail"
      />
      <input
        name="senha"
        type="password"
        value="Senha"
      />
      <input
        name="telefone"
        value="Telefone"
      />
      <button
        type="submit"
      >
        Cadastre-se
      </button>
    </form>
  </div>
</DocumentFragment>
`;
